client/my-sites/checklist/wpcom-checklist/wpcom-task-list.js

```diff
@@ -31,7 +31,8 @@ function getTasks( {
 	const tasks = [];
 	const segmentSlug = getSiteTypePropertyValue( 'id', siteSegment, 'slug' );
 
-	const getTask = taskId => get( taskStatuses, taskId );
+	const getTask = taskId =>
+		taskStatuses ? taskStatuses.filter( task => task.id === taskId )[ 0 ] : undefined;
 	const hasTask = taskId => getTask( taskId ) !== undefined;
 	const isCompleted = taskId => get( getTask( taskId ), 'completed', false );
 	const addTask = ( taskId, completed ) => {
```

Comment:
Works for me with `isCompleted` 

`const isCompleted = taskId => get( getTask( taskId ), 'isCompleted', false );`

Reply:
@sirreal Does that work for https://github.com/Automattic/wp-calypso/pull/34050#issuecomment-503144833 ?

Reply:
Another task for our backlog: add unit tests to `tasks.js` :)

Reply:
I think you're correct but haven't verified. Strange that the Jetpack checklist seemed to work for me 😕